data/lib/src/network/linshare_http_client.dart

```diff
@@ -493,9 +493,6 @@ class LinShareHttpClient {
     final resultJson = await _dioClient.put(
       Endpoint.sharedSpaces.withPathParameter(sharedSpaceId.uuid).generateEndpointPath(),
       data: renameWorkGroupBodyRequest.toJson().toString(),
-      options: Options(headers: {
-        'Content-Type': 'application/json; charset=UTF-8'
-      })
```

Comment:
Not because of it. Because it misses the nodeType parameter on v4.0 only. I have checked `Content-Type` here which is not necessary because it is already set for us by default. So I deleted it to avoid misunderstandings.